browser/src/screenshot.ts

```diff
@@ -1,23 +1,38 @@
 import { type Page } from 'playwright'
 import * as gptscript from '@gptscript-ai/gptscript'
-import { getWorkspaceId } from './session.ts'
+import { getWorkspaceId, getGPTScriptEnv } from './session.ts'
 import { type IncomingHttpHeaders } from 'node:http'
 
 const client = new gptscript.GPTScript();
+const ottoServerUrl = process.env.OTTO_SERVER_URL
```

Comment:
The environment variable is set the same way the `OTTO_THREAD_ID` variable is. So, I would expect that "getting" it would be the same too.

Reply:
Hmm, this is working locally, and I wouldn't expect it to be different when returned via the header, so is there any reason to get it from the header on every request instead of once at daemon startup?

Reply:
You've convinced me. Code updated.